proposals/0047-last-hard-fork-syscall.md

```diff
@@ -0,0 +1,120 @@
+---
+simd: '0047'
+title: Syscall to get the last hardfork
+authors:
+  - Godmode Galactus (Mango Markets)
+category: Standard
+type: Core
+status: Draft
+created: 2023-04-15
+---
+
+## Summary
+
+Create a new sysvar `SysvarLastHardFork1111111111111111111111111`, which can
+be used to get the last hard fork slot.
+
+## Motivation
+
+In Solana, when the cluster cannot reach consensus, it is restarted using
+`ledger-tool` with a hard fork on a slot. A hard fork is usually done on an
+optimistically confirmed slot which was voted on and accepted by 50% of the
```

Comment:
This definition of optimistically confirmed slot  is incorrect. I suggest removing the definition entirely since this SIMD is not introducing the term